### ISSUE TITLE
[Spell interaction] MARK OF BLOOD under Mind Control in ICC25hc heals LDW for 18-19 Million

### ISSUE THREAD

**Issue by MailOrdered** [WoWManiaUK/Redemption#1744]
_

> I recently wiped on a Lady Deathwhisper in ICC 15 heroic because our Tank paladin healed the boss for 15-18 Million in one Lay on Hands while under Mind Control. I'm not sure if this is a bug or is possible with certain buffs (sure as hell doesn't feel like it).
> 
> Cueball is the tank pally. In full raid buffs (looking after the wipe) he has 62327 hp. I'm not sure what the math is on Mind Control healing bonuses so I can't check myself and am not sure if this is possible without a bug. Reporting it here anyways.

_

**SEE BELOW**

Skada reports 18.89 million healing, but we had been combat bugged for I think 2 or 3 pulls to mid p2, so the numbers are a little messy but he's the tank so it shouldn't change it that much for his numbers.

Proof:
![wowscrnshot_101517_185335](https://user-images.githubusercontent.com/29909872/31591456-264eb08a-b1dd-11e7-975c-2223045bb9f6.jpg)

**Comment by MailOrdered** [comment]
Also: Sada reports that he use "Mark of Blood" to do his healing. I have no idea what this is and it has a shaman icon.

![wowscrnshot_101517_195308](https://user-images.githubusercontent.com/29909872/31592135-57a14e26-b1e3-11e7-98e1-07b12c0c0f24.jpg)

**Comment by MailOrdered** [comment]
http://www.wowhead.com/spell=206940/mark-of-blood

One of our Dks has this talent, Is this working as intended? WAAAY too much healing.

**Comment by squalllh1989** [comment]
I'm not sure what the situation happened, but on my side, i think it should happen like this: the dk that has [Mark of Blood](http://www.wow-mania.com/armory/?spell=49005) got mind controled, he used it on the paladin (your tank). The Mark of Blood work like this: when the dk used the mark on target A, if target A hit target B, each hit of A will heal B for 4% of B's **max HP**, for maximum of 20 second duration or 20 hit, if he hit max 20 hit, it's 20 x 4 = 80% **max HP**. Lady Deathwhisper in 25hc has 13,624,800 HP, so if the one got the mark hit for the max hit posible when get the mark, he will heal Lady Deathwhisper for 10,899,840 HP. You said that you were stuck in combat for 2-3 pull, so it's totally posible, if the dk that got the mark being mind controlled for at least 2 pull and used his mark on the paladin. We have suffered a situation like this once, when we got lady to below 10% hp, then she mind control one of our tank, which is a blood dk, he used his Mark of Blood on a hunter, and the hunter keep shot at Lady Deathwhisper with all of his posible might and speed, and heal her up to more than 80% hp, which caused the wipe. I didn't tested this yet so i can't say that the healing of the mark is 100% correct, but yes, it is working as intended. I will close it for now, if you still have a trouble with this, feel free to comment below.
